frontend/packages/core/src/AppLayout/drawer.tsx

```diff
@@ -1,198 +1,213 @@
   Drawer as MuiDrawer,
-  Grid,
   List,
   ListItem,
   ListItemText,
+  Paper as MuiPaper,
+  Popper as MuiPopper,
   Typography,
 } from "@material-ui/core";
-import ExpandLess from "@material-ui/icons/ExpandLess";
-import ExpandMore from "@material-ui/icons/ExpandMore";
-import styled from "styled-components";
 
-import { useTheme } from "../AppProvider/themes";
 import { useAppContext } from "../Contexts";
-import { TrendingUpIcon } from "../icon";
 
-import Logo from "./logo";
-import { userId } from "./user";
 import { routesByGrouping, sortedGroupings } from "./utils";
 
-const mobileDrawerWidth = "90%";
-const drawerWidth = "300px";
-
-const DrawerPanel = styled(MuiDrawer)`
-  ${({ theme }) => `
-  flex-shrink: 0;
-  min-width: ${drawerWidth};
-  @media screen and (max-width: 800px) {
-    min-width: ${mobileDrawerWidth};
-  }
-  div[class*="MuiDrawer-paper"] {
-    min-width: ${drawerWidth};
-    background-color: ${theme.palette.secondary.main};
-    padding: ${theme.spacing(2)}px;
-    @media screen and (max-width: 800px) {
-      min-width: ${mobileDrawerWidth};
-    }
-  }
-  `}
-`;
-
-const DrawerHeader = styled(Grid)`
-  ${({ theme }) => `
-    justify: flex-start;
-    direction: row;
-    ${theme.mixins.toolbar};
-  `}
-`;
-
-const ExpandLessIcon = styled(ExpandLess)`
-  ${({ theme }) => `
-  color: ${theme.palette.primary.main};
-  `}
-`;
-
-const ExpandMoreIcon = styled(ExpandMore)`
-  ${({ theme }) => `
-  color: ${theme.palette.primary.main};
-  `}
-`;
-
-const GroupHeading = styled(Typography)`
-  ${({ theme }) => `
-  color: ${theme.palette.primary.main};
-  padding-top: 0.25rem;
-  font-weight: bolder;
-  `}
-`;
-
-const TrendingIcon = styled(TrendingUpIcon)`
-  fontsize: 20px;
-  marginleft: 10px;
-`;
-
-const NavigationLink = styled(RouterLink)`
-  ${({ theme }) => `
-  color: ${theme.palette.text.secondary};
-  `}
-`;
-
-interface HeaderProps {
-  onNavigate: () => void;
-}
-
-const Header: React.FC<HeaderProps> = ({ onNavigate }) => {
-  return (
-    <DrawerHeader container flex-wrap="wrap">
-      <Grid item>
-        <NavigationLink to="/" onClick={onNavigate} data-qa="logo">
-          <Logo />
-        </NavigationLink>
-      </Grid>
-      <Grid item>
-        <Grid container justify="center" direction="column">
-          <Typography
-            style={{ lineHeight: "1.3", margin: "-5px 0px 10px 10px" }}
-            component="span"
-            color="primary"
-            data-qa="title"
-          >
-            <NavigationLink to="/" onClick={onNavigate}>
-              <GroupHeading>clutch</GroupHeading>
-            </NavigationLink>
-            <Typography variant="caption">Welcome {userId()}!</Typography>
-          </Typography>
-        </Grid>
-      </Grid>
-    </DrawerHeader>
-  );
-};
-
-const Divider = styled(MuiDivider)`
-  ${({ theme }) => `
-  background-color: ${theme.palette.accent.main};
-  `}
-`;
+// sidebar
+const drawerWidth = "100px";
+
+const DrawerPanel = styled(MuiDrawer)({
```

Comment:
i removed the mobile drawer width b/c the design isn't mobile friendly with a sidebar popper. for mobile, we'd probably want the old design.